xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/less/variablesInit.vm

```diff
@@ -265,9 +265,9 @@
   ## from an orangish yellow to complete brown. Instead, we change the text color to black.
   @btn-warning-color:                   #000;
   ## The boxes should have enough contrast by default, we make it lighter than bootstrap.
-  @state-danger-bg:                     #f8ecec;
-  @state-success-bg:                    #e5f3df;
-  @state-info-bg:                       #e1f1f9;
+  @alert-danger-bg:                     #f8ecec;
+  @alert-success-bg:                    #e5f3df;
+  @alert-info-bg:                       #e1f1f9;
```

Comment:
I'm somehow missing "warning" in this list. According to the styles above, there should have been a similar `@state-warning-bg` variable. Was it missing before?

Reply:
It wasn't a violating value from bootstrap, so there was no need to update it :)
Maybe I should add a comment here?

Reply:
Added a comment in https://github.com/xwiki/xwiki-platform/pull/2897/commits/e1e311523b8bf28dc8bbacf365b0143aa418555c :+1: